system-variables.md

```diff
@@ -643,7 +678,7 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 ### tidb_gc_life_time <span class="version-mark">New in v5.0</span>
 
 - Scope: GLOBAL
-- Default: `"10m0s"`
+- Default value: 10m0s
```

Comment:
This value was presented as a "string". If so, I think we can keep the "" quotes. The same for similar cases in this PR.

Reply:
If we change this to a string, then the other cases would need to be a string too. It's not that this is incorrect - it's just that it doesn't add any value (all sysvars are actually intepreted as strings). See: https://dev.mysql.com/doc/refman/5.7/en/innodb-parameters.html#sysvar_innodb_flush_method for example on how mysql does it.